src/core/mac/mac_frame.cpp

```diff
@@ -1061,49 +1061,37 @@ const TimeIe *Frame::GetTimeIe(void) const
 #if OPENTHREAD_CONFIG_MULTI_RADIO
 uint16_t Frame::GetMtu(void) const
 {
-    uint16_t mtu;
+    uint16_t mtu = OT_RADIO_FRAME_MAX_SIZE;
 
     switch (GetRadioType())
     {
-#if OPENTHREAD_CONFIG_RADIO_LINK_IEEE_802_15_4_ENABLE
-    case kRadioTypeIeee802154:
-        mtu = OT_RADIO_FRAME_MAX_SIZE;
-        break;
-#endif
-
 #if OPENTHREAD_CONFIG_RADIO_LINK_TREL_ENABLE
     case kRadioTypeTrel:
         mtu = Trel::Link::kMtuSize;
         break;
 #endif
 
     default:
-        OT_ASSERT(false);
+        break;
     }
 
     return mtu;
 }
 
 uint8_t Frame::GetFcsSize(void) const
 {
-    uint8_t fcsSize;
+    uint8_t fcsSize = k154FcsSize;
```

Comment:
Fix: `error: ‘fcsSize’ may be used uninitialized in this function [-Werror=maybe-uninitialized]`